backend/FwLite/LocalWebApp/LocalWebApp.csproj

```diff
@@ -47,4 +47,9 @@
         <EmbeddedResource Include="..\..\..\frontend\viewer\dist\**\*" />
     </ItemGroup>
 
+    <ItemGroup>
+        <Content Include="/opt/local/lib/libicu*.??.dylib" Condition="$([MSBuild]::IsOsPlatform('macOS'))">
```

Comment:
This assumes that icu4c libs are installed under "/opt/local/lib" on macOS. This is the default location for libs that MacPorts installs. If we need to start bundling icu4c libs from somewhere other than "/opt/local/lib", this path and some other paths in the build scripts will need to be updated. We have to bundle the icu4c libs somehow unless macOS starts to include them by default.